packages/dev/parcel-transformer-icon/IconTransformer.js

```diff
@@ -33,6 +37,12 @@ module.exports = new Transformer({
                   inlineStyles: {
                     onlyMatchedOnce: false,
                     removeMatchedSelectors: true
+                  },
+                  cleanupIDs: {
+                    prefix
+                  },
+                  convertPathData: {
+                    makeArcs: false
```

Comment:
Adds a hash prefix to all id attributes so they don't conflict when multiple illustrations are on the page at the same time. Also disables `makeArcs` because it was breaking one of the linear illustrations.